.vscode/settings.json

```diff
@@ -0,0 +1,4 @@
+{
```

Comment:
I know this is all just WIP, but can you eventually please remove this file from the PR?

Reply:
sure